crates/pathfinder/src/sequencer.rs

```diff
@@ -89,6 +89,19 @@ pub trait ClientApi {
         contract_definition: ContractDefinition,
         token: Option<String>,
     ) -> Result<reply::add_transaction::DeployResponse, SequencerError>;
+
+    #[allow(clippy::too_many_arguments)]
+    async fn add_deploy_account(
+        &self,
+        version: TransactionVersion,
+        max_fee: Fee,
+        signature: Vec<TransactionSignatureElem>,
+        nonce: TransactionNonce,
+        contract_address_salt: ContractAddressSalt,
+        class_hash: ClassHash,
+        calldata: Vec<CallParam>,
+        token: Option<String>,
```

Comment:
We should clarify with Ariel if the `token` is also something we have to support here.

It was basically a hack to support permissioned deploys on mainnet -- I'm not sure it's still required.

Reply:
Good point, there is no token in API dump. Removing.

Reply:
For DEPLOY_ACCOUNT no token is sent by the CLI anyway, so I'm removing it for now. It is super easy to restore if necessary anyway.